africanus/conftest.py

```diff
@@ -1,12 +1,18 @@
 # -*- coding: utf-8 -*-
 
+import os
+
 from numba.core.runtime import rtsys
 import pytest
 
 from africanus.util.testing import mark_in_pytest
 
 
-@pytest.fixture(scope="function", autouse=True)
+NUMBA_NRT_STATS_ENABLED = (
```

Comment:
Is this the best way of doing this? It should be possible to do `numba.config.NRT_STATS`, but not sure of the context here.

Reply:
Yes, thats much nicer, thanks for pointing that out.